docs/install_guides/pypi-openvino-dev.md

```diff
@@ -99,6 +99,7 @@ For example, to install and configure the components for working with TensorFlow
    ```sh
    pip install openvino-dev[tensorflow2,mxnet,caffe]
    ```
+**NOTE**: Support of MO in TensorFlow 1.x environment is deprecated. Use TensorFlow 2.x environment to convert both TensorFlow 1.x and 2.x models
```

Comment:
should we only have `pip install openvino-dev[tensorflow]` version for TF? Or `tensorflow2` is kept for backward compatibility?

Reply:
We decide to have both for backward compatibility.

Reply:
will we have them both forever?

Reply:
I don't know. I think we need to adress this question to appropriate person.

Reply:
Anyway we keep them for next release.

Reply:
Let's proceed with both requirements files for the upcoming release